modules/bot/test/github-client.spec.ts

```diff
@@ -217,9 +213,11 @@ describe('github-client', () => {
             payload: payloadFixture,
           } as any);
 
+          expect(mockCompleteJob).toHaveBeenCalledWith('my-job-id');
```

Comment:
Use `id` instead of `'my-job-id'` here to connect it back to the const variable above

```suggestion
          expect(mockCompleteJob).toHaveBeenCalledWith(id);
```